### PR TITLE
Clarification for bundle build instructions

### PR DESCRIPTION
**Pull request by duncan-brown**

### CODE REVIEW

File: docs/building_bundled_executables.rst [hunk 1]
```diff
@@ -81,14 +96,29 @@ GitHub repository ior branch you can use the arguments::
...
+passing the argument::
+
     --with-extra-bank=/home/soumi.de/projects/cbc/SEOBNRROM-proj/testbank_TF2v4ROM.hdf
 
+.. note::
+
+    Command line parsing is performed with shell syntax matching, not Python
+    argument parsing, so command line arguments must be given as
+    ``--option=argument`` with the option name spelled out in full and including
+    the ``=`` betwwn the option and the argument.
```
Comment (bema-ligo) [inline docs/building_bundled_executables.rst]: Small typo here

File: docs/building_bundled_executables.rst [hunk 2]
```diff
@@ -47,10 +47,25 @@ software installed.
...
 The command-line arguments for the ``pycbc_build_eah.sh`` build script are:
 
 .. command-output:: pycbc_build_eah.sh --force-debian4 --help
 
+The build script creates the directories ``pycbc-sources`` and ``pycbc-build``
+in the directory that it is run from.  The build script creates the bundles in
+the directory ``pycbc-build/environment/dist/`` relative to the directory from
+which it run. The ``pycbc_inspiral`` executable will be named
+``pycbc_inspiral_osg`` or ``pycbc_inspiral_osg_TAG`` if the git commit matches
+a tag name ``TAG`` in the repository.
```
Comment (bema-ligo) [inline docs/building_bundled_executables.rst]: Actually not for an arbitrary tag, but only if the tag has the format of a revision number `^\.v[0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*$'`

Comment (duncan-brown) [inline docs/building_bundled_executables.rst]: Thanks, fixed.

File: docs/building_bundled_executables.rst (hunk 1 above)
Comment (duncan-brown) [inline docs/building_bundled_executables.rst]: Fixed and moved up the page to just after the ``--help`` message.

**Comment by duncan-brown** [comment]
@bema-ligo thanks, I've addressed the comments. Please can you approve?

**Pull request merged**